QUARKUS-1164.md

```diff
@@ -0,0 +1,26 @@
+# Tech Preview Support for Oracle JDBC Extension
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1164
+https://quarkus.io/guides/datasource
+
+Oracle datasource using `jdbc-oracle` extension.
+
+## Scope of testing
+- Connect to a database and persist data using Hibernate ORM with Panache.
+
+### Existing tests
+- `jpa-oracle` module in Quarkus upstream: https://github.com/quarkusio/quarkus/tree/main/integration-tests/jpa-oracle
+  - CRUD operations.
+  - Invocation of a DB procedure.
+
```

Comment:
Add link to the analysis done on the RFE  - https://issues.redhat.com/browse/QUARKUS-1164?focusedCommentId=18935716&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18935716